### DISCUSSION TITLE
Senso Chromebook

### DISCUSSION THREAD

**Discussion by Lytonicity** [3kh0/ext-remover#775] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
So I was wondering if there is an exploit for the Senso Extension.....Basically IT can see what your are doing on your screen, disable your Wi-Fi, and close tabs and etc. Bookmarklets are currently disable on my school Chromebook too. Is there any exploit out that can disable it?

**Comment by gooseisgoose** [comment]
get a extension that disables extensions. if that doesn't work then ur fucked.

**Comment by kennethfailehood-oss** [comment]
i founda way to do it at my school i just use site perms
